hw/i386/virt/acpi.c

```diff
@@ -48,6 +48,9 @@ typedef struct VirtAcpiState {
     AcpiPciHpState pcihp_state;
```

Comment:
@liujing2 we should make this a list longer term.

Reply:
@mcastelino Yes. One problem about pcihp_state is how to distinguish which pcihp_state[i] to give in [virt_device_plug_cb](https://github.com/intel/nemu/blob/857d652a0faccfbf61d33ac4efbca8b1581c9bae/hw/i386/virt/acpi.c#L80), and this seems similar to the problem of PCNT.